test/new-e2e/pkg/utils/e2e/client/docker.go

```diff
@@ -128,20 +105,8 @@ func (docker *Docker) GetClient() *client.Client {
 	return docker.client
 }
 
-// GetAgentContainerName gets the agent container name
-func (docker *Docker) GetAgentContainerName() string {
-	require.NotEmptyf(docker.t, docker.agentContainerName, "agent container not found")
-	return docker.agentContainerName
-}
-
-// GetAgent gets an instance that implements the [Agent] interface. This function panics, if there is no agent container.
-func (docker *Docker) GetAgent() Agent {
-	require.NotNilf(docker.t, docker.optionalAgent, "there is no agent installed on this docker instance")
-	return docker.optionalAgent
-}
-
-func (docker *Docker) executeAgentCmdWithError(commands []string) (string, error) {
-	wholeCommands := []string{"agent"}
-	wholeCommands = append(wholeCommands, commands...)
-	return docker.ExecuteCommandWithErr(docker.GetAgentContainerName(), wholeCommands...)
-}
+// func (docker *Docker) executeAgentCmdWithError(commands []string) (string, error) {
+// 	wholeCommands := []string{"agent"}
+// 	wholeCommands = append(wholeCommands, commands...)
+// 	return docker.ExecuteCommandWithErr(docker.GetAgentContainerName(), wholeCommands...)
+// }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
```